### PR TITLE
Adding cert-manager crds

### PR DESCRIPTION
**Pull request by origranot**

### CODE REVIEW

File: charts/global-templates/templates/_certificate.yaml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+{{- define "global-templates.certificate" -}}
+{{- $files := .Files }}
+{{- $global_values := .Values.global }}
+{{- $default_values := .Values.certificate_defaults }}
+{{- $common_defaults := .Values.common_defaults | default dict }}
+{{- range .certificates }}
+{{- if not .disabled }}
+{{- $certificate := . }}
+{{- if not (hasKey . "secretName") }}
```
Comment (origranot) [inline charts/global-templates/templates/_certificate.yaml:14]: Use $certificate here (correct for all files, all places)

File: charts/global-templates/templates/_certificate.yaml [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  {{- if $certificate.ipAddresses }}
+  ipAddresses:
+    {{- range $certificate.ipAddresses }}
+    - {{ include "helpers.renderGlobalIfExists" (dict "value" . "global" $global_values) }}
+    {{- end }}
+  {{- end }}
+  {{- if $certificate.uris }}
+  uris:
+    {{- range $certificate.uris }}
+    - {{ include "helpers.renderGlobalIfExists" (dict "value" . "global" $global_values) }}
```
Comment (origranot) [inline charts/global-templates/templates/_certificate.yaml:71]: Nice!

File: charts/global-templates/templates/_clusterissuer.yaml [hunk 3]
```diff
@@ -0,0 +1,62 @@
+{{/*
+Copyright (C) 2024 XM Cyber
```
Comment (origranot) [inline charts/global-templates/templates/_clusterissuer.yaml:2]: 2025

File: charts/global-templates/templates/_certificate.yaml [hunk 4]
```diff
@@ -0,0 +1,107 @@
+{{/*
+Copyright (C) 2024 XM Cyber
```
Comment (origranot) [inline charts/global-templates/templates/_certificate.yaml:2]: 2025

File: charts/test-chart/tests/certificate/certificate_test.yaml [hunk 5]
```diff
@@ -0,0 +1,38 @@
+suite: Test certificate template
```
Comment (origranot) [inline charts/test-chart/tests/certificate/certificate_test.yaml:1]: Consider add more edge cases (same for all new crds)

**Review by origranot — COMMENTED** [review]
Looks good, I left few comments.
In addition, please add docs about the resources you added.
https://github.com/XMCyber/helm-global-templates/tree/master/docs/resources

File: charts/global-templates/templates/_certificate.yaml (hunk 1 above)
Comment (Danielkiss9) [reply charts/global-templates/templates/_certificate.yaml]: Done

File: charts/global-templates/templates/_certificate.yaml (hunk 2 above)
Comment (Danielkiss9) [reply charts/global-templates/templates/_certificate.yaml:71]: thx <3

File: charts/global-templates/templates/_clusterissuer.yaml (hunk 3 above)
Comment (Danielkiss9) [reply charts/global-templates/templates/_clusterissuer.yaml]: Done

File: charts/global-templates/templates/_certificate.yaml (hunk 4 above)
Comment (Danielkiss9) [reply charts/global-templates/templates/_certificate.yaml]: Done

File: charts/test-chart/tests/certificate/certificate_test.yaml (hunk 5 above)
Comment (Danielkiss9) [reply charts/test-chart/tests/certificate/certificate_test.yaml:1]: Done
